extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -74,6 +75,39 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 			}
 		} catch (e) {
 			vscode.window.showErrorMessage(loc.openFileError(bookPath, e instanceof Error ? e.message : e));
+		} finally {
```

Comment:
If we couldn't open the book, then we shouldn't be executing this code. This probably should be in the try.